src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -96,17 +96,16 @@ public IEnumerable<PopulationPKAnalysis> CalculateFor(IPopulationDataCollector p
          if (timeProfileChartData == null)
             return pkAnalyses; // there are no analyses to calculate
 
-         var allColumns = timeProfileChartData.Panes.SelectMany(x => x.Curves).Select(x =>
-               new {curveData = x, column = columnFor(x, populationDataCollector)})
-            .Where(c => c.column != null)
+         var allColumns = timeProfileChartData.Panes.SelectMany(x => x.Curves).SelectMany(x =>
```

Comment:
I don't understand this change.

Reply:
It refers to range results. Before range results only contained a single result by now it is calculated at both ends so the 5 and 95 % for example instead of 5% only.